corehq/messaging/scheduling/forms.py

```diff
@@ -2966,7 +2966,8 @@ def add_additional_content_types(self):
             self.fields['content'].choices += [
                 (self.CONTENT_CUSTOM_SMS, _("Custom SMS")),
             ]
-        if settings.FCM_CREDS:
+
+        if settings.FCM_CREDS or self.initial.get('content') == self.CONTENT_FCM_NOTIFICATION:
```

Comment:
should this be and?

Reply:
I added this or condition for a rare case where FCM was removed from the environment, This causes the existing alerts with Push Notification as Content to pick up  as `SMS` in the UI.

Let me know if you think this scenario is not practical and I can get rid of this.

Reply:
> This causes the existing alerts with Push Notification as Content to pick up as SMS in the UI.
oh, that is not nice. We could have the dropdown to be blank and not SMS, is blank an option? Or block editing at all because all other options related to FCM will also become invalid, so a user should only be able to delete it?

This would cause all notifications to fail or they won't get trigered only?

Reply:
`blank` is not an option. 
Ideally any existing conditional alerts itself should be marked as `inactive`  or else it would try to send notifications which will fail with an error logged `MessagingEvent.ERROR_FCM_NOT_AVAILABLE` .

I think this would be applicable also if a domain removed from this feature flag has existing alerts. 

I see not similar however a close use case with `Emails` where they only log the error while sending the email.

Reply:
> Ideally any existing conditional alerts itself should be marked as inactive

I like this, we could take this as a follow up to this PR so we are don't block this one but we should definitely do something.

Reply:
Added a ticket to track this: https://dimagi-dev.atlassian.net/browse/INDIV-174